### PR TITLE
Showing text, when no events available

### PR DESCRIPTION
**Pull request by Shivam-Katare**
## Fixes Issue

#2168

## Changes proposed

I've added text content that will appear only when no events are available.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot (292)

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2195"></a>

### CODE REVIEW

File: pages/events.js [hunk 1]
```diff
@@ -27,13 +27,18 @@ export default function Events({ events }) {
...
-        <ul>
-          {events.map((event) => (
-            <li key={event.name}>
-              <EventPreview event={event} username={event.author} />
-            </li>
-          ))}
-        </ul>
+        {!events.length
+          ?
+          <h3 className="text-center text-4xl mt-10 font-mono"> No Events Available</h3>
```
Comment (schmelto) [inline pages/events.js:32]:
```suggestion
          <h3 className="text-center text-4xl mt-10 font-mono">No events available.</h3>
```

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

---
**Summary:** :ok_hand: This code looks good

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Comment by eddiejaoude** [comment]
It was discussed in the issue to use an alert similar to the search page to be consistent, this can be fixed in another PR though

**Pull request merged**